app/client/cypress/integration/state/water-quality-overview/subTab.spec.js

```diff
@@ -0,0 +1,66 @@
+describe('Water Quality Overview Sub Tabs', () => {
+  beforeEach(() => {
+    cy.visit('/state/FL/water-quality-overview');
+  });
+
+  it('Navigating to a sub-tab selection that has no data results in “Water Type” dropdown saying “No Available Water Types” and the “Use” dropdown saying “No Available Uses”', () => {
+    // verify the water quality overview content loaded prior to other tests
+    cy.findByText('Water Quality', { timeout: 20000 }).should('exist');
+
+    // wait for the all web services to finish (surveys is usually slow here)
+    cy.findAllByTestId('hmw-loading-spinner', { timeout: 20000 }).should(
+      'not.exist',
+    );
+
+    // verify a tab without data
+    cy.findByTestId('hmw-swimming-tab-button').click();
+    cy.findByTestId('hmw-swimming-tab-panel').contains(
+      'No Available Water Types',
+    );
+    cy.findByTestId('hmw-swimming-tab-panel').contains('No Available Uses');
+
+    // verify a tab with data
+    cy.findByTestId('hmw-fishing-tab-button').click();
+    cy.findByTestId('hmw-fishing-tab-panel')
+      .contains('No Available Water Types')
+      .should('not.exist');
+    cy.findByTestId('hmw-fishing-tab-panel')
+      .contains('No Available Uses')
+      .should('not.exist');
+  });
+
+  it('Navigating to a sub-tab selection shows correct charts', () => {
+    // verify the water quality overview content loaded prior to other tests
+    cy.findByText('Water Quality', { timeout: 20000 }).should('exist');
+    cy.findByTestId('hmw-ecological-tab-button').click();
+
+    // wait for the all web services to finish (surveys is usually slow here)
+    cy.findAllByTestId('hmw-loading-spinner', { timeout: 20000 }).should(
+      'not.exist',
+    );
```

Comment:
(Same comment from other test... if you move this up to the `beforeEach()` block, remove it from this test).